src/safeposix/syscalls/fs_calls.rs

```diff
@@ -3650,7 +3690,7 @@ impl Cage {
                                 // convert filename to a filename vector of u8
                                 let mut vec_filename: Vec<u8> = filename.as_bytes().to_vec();
                                 vec_filename.push(b'\0'); // make filename null-terminated
-
+                                // Push DT_UNKNOWN as d_type. This is a placeholder for now, as the actual file type is not yet determined.
```

Comment:
some weird whitespace here, have you done a cargo fmt?

Reply:
yes even with cargo fmt the whitespace is there